internal/handlers/common/projection.go

```diff
@@ -18,6 +18,17 @@ import "github.com/FerretDB/FerretDB/internal/types"
 
 // ProjectDocuments modifies given documents in places according to the given projection.
 func ProjectDocuments(docs []*types.Document, projection *types.Document) error {
-	// TODO
+	if projection.Len() == 0 {
+		return nil
+	}
+	projectionMap := projection.Map()
+	for i := 0; i < len(docs); i++ {
+		for k := range docs[i].Map() {
+			if _, ok := projectionMap[k]; !ok {
```

Comment:
We should check the value too. For example, 0 or false should remove the field.

Or the plan is to do that in a different PR?

Reply:
Then I'd add new tests for it too.

Reply:
I'd rather add tests and add code to comply with https://docs.mongodb.com/manual/reference/method/db.collection.find/#projection

Reply:
In this PR, or in different one?

Reply:
In this PR